File: src/Net/1. First Course/5. Fourth Hole/RefactoringGolf.Store/CloathingDiscount.cs
namespace RefactoringGolf.Stack
{
    public class CloathingDiscount : ICategoryDiscount
    {
        public decimal CalculateDiscount(OrderItem orderItem)
        {
            decimal discount = 0;
            if (orderItem.Quantity > 2)
            {
                discount = orderItem.Product.UnitPrice;
            }
            return discount;
        }
    }
}

File: src/Java/1. First Course/3. Third Hole/src/main/refactoringgolf/store/CloathingDiscount.java
package refactoringgolf.store;

public class CloathingDiscount implements CategoryDiscount {

	public float calculateDiscount(OrderItem orderItem) {
		float discount = 0;
		if (orderItem.getQuantity() > 2) {
			discount = orderItem.getProduct().getUnitPrice();
		}
		return discount;
	}

}
